src/setup/devnet/index.js

```diff
@@ -477,9 +560,18 @@ export class Devnet {
                         "devnet",
                     ];
 
-                    // create heimdall folder
-                    shell.exec(`sudo mkdir -p /var/lib/heimdall`)
-                    shell.exec(`sudo chmod 777 -R /var/lib/heimdall/`)
+                    // create heimdall folder for all the nodes
```

Comment:
Can we move this step to a later phase where we already iterate over nodes?
This would avoid a double iteration.
Maybe we can move it under
`// set heimdall peers with devnet heimdall hosts`
Also, if `execa` does not work with `mkdir` (because `mkdir` is considered another file, different from `ssh`), you can use `shell.exec`